package.json

```diff
@@ -7,7 +7,7 @@
     "@blueprintjs/datetime": "~3.13.0",
     "@blueprintjs/icons": "~3.10.0",
     "@terralego/core": "~1.6.5",
-    "bootstrap": "^4.3.1",
+    "bootstrap": "^3.3.7",
```

Comment:
Any reason for rolling back bootstrap version in the PR?

Reply:
Every component from react-jsonschema-form works fine with the ui from bootstrap 3.3.7.
At the beginning, i tried with the last version but there are too many components to rework in CSS.

One of the core developper of RJSF started and issue about a support of bootstrap 4 https://github.com/rjsf-team/react-jsonschema-form/issues/1455